cdap-cli-tests/src/test/java/io/cdap/cdap/cli/AuthorizationCLITest.java

```diff
@@ -141,75 +141,79 @@ public void testAuthorizationCLI() throws Exception {
 
-    testCommandOutputContains(cli, String.format("create namespace %s", namespaceId.getNamespace()),
+    CLITestBase.testCommandOutputContains(cli, String.format("create namespace %s", namespaceId.getNamespace()),
                               String.format("Namespace '%s' created successfully", namespaceId.getNamespace()));
 
     // test creating role
-    testCommandOutputContains(cli, "create role " + role.getName(), String.format("Successfully created role '%s'",
-                                                                                  role.getName()));
+    CLITestBase.testCommandOutputContains(cli, "create role " + role.getName(),
+                                          String.format("Successfully created role '%s'",
+                                                        role.getName()));
 
     // test add role to principal
-    testCommandOutputContains(cli, String.format("add role %s to %s %s", role.getName(), principal.getType(),
-                                                 principal.getName()),
-                              String.format("Successfully added role '%s' to '%s' '%s'", role.getName(),
-                                            principal.getType(), principal.getName()));
+    CLITestBase.testCommandOutputContains(cli, String.format("add role %s to %s %s", role.getName(),
+                                                             principal.getType(),
+                                                             principal.getName()),
+                                          String.format("Successfully added role '%s' to '%s' '%s'", role.getName(),
+                                                        principal.getType(), principal.getName()));
 
     // test listing all roles
-    String output = getCommandOutput(cli, "list roles");
+    String output = CLITestBase.getCommandOutput(cli, "list roles");
     List<String> lines = Arrays.asList(output.split("\\r?\\n"));
     Assert.assertEquals(2, lines.size());
     Assert.assertEquals(role.getName(), lines.get(1)); // 0 is just the table headers
 
     // test listing roles for a principal
-    output = getCommandOutput(cli, String.format("list roles for %s %s", principal.getType(), principal.getName()));
+    output = CLITestBase.getCommandOutput(cli, String.format("list roles for %s %s", principal.getType(),
+                                                             principal.getName()));
     lines = Arrays.asList(output.split("\\r?\\n"));
     Assert.assertEquals(2, lines.size());
     Assert.assertEquals(role.getName(), lines.get(1));
 
     // test grant action. also tests case insensitivity of Action and Principal.PrincipalType
-    testCommandOutputContains(cli, String.format("grant actions %s on entity %s to %s %s",
+    CLITestBase.testCommandOutputContains(cli, String.format("grant actions %s on entity %s to %s %s",
                                                  Action.READ.name().toLowerCase(), namespaceId.toString(),
```

Comment:
Try to keep the same alignment with the "String.fornat(" block.. same for everywhere else.

Reply:
Is there a setting for Intellij to autoformat on save?

Reply:
Reformatted with CDAP settings